config/passport.js

```diff
@@ -101,3 +102,12 @@ var linkedinStrategy = new LinkedinStrategy({
   done(null, formatProfile(profile));
 });
 passport.use(linkedinStrategy);
+
+var linkedinTokenStrategy = new LinkedInTokenStrategy({
+  clientID: process.env.LINKEDIN_API_KEY,
+  clientSecret: process.env.LINKEDIN_API_SECRET,
+  callbackURL: url('/noo/login/linkedin/oauth'),  
```

Comment:
actually, it doesn't need to be there at all. Removed.

On Wed, Jul 1, 2015 at 3:59 PM, Lawrence Wang <notifications@github.com>
wrote:

> In config/passport.js
> <https://github.com/Hylozoic/hylo-node/pull/28#discussion_r33733878>:
>
> > @@ -101,3 +102,12 @@ var linkedinStrategy = new LinkedinStrategy({
> >    done(null, formatProfile(profile));
> >  });
> >  passport.use(linkedinStrategy);
> > +
> > +var linkedinTokenStrategy = new LinkedInTokenStrategy({
> > +  clientID: process.env.LINKEDIN_API_KEY,
> > +  clientSecret: process.env.LINKEDIN_API_SECRET,
> > +  callbackURL: url('/noo/login/linkedin/oauth'),
>
> i think you meant to put a -token here
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Hylozoic/hylo-node/pull/28/files#r33733878>.
>



-- 
Robbie Carlton

719.358.3482
<robbie@authenticmanprogram.com>